profile/Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org/"
+
+gem "rake"
+gem "ruby-prof"
```

Comment:
Slight difference from before, we'll allow any version of rake and ruby-prof now. Possibly not a bad thing though.